### PR TITLE
Fix MSVC build

### PR DESCRIPTION
**Pull request by dnachbaur**

### CODE REVIEW

**Comment by dnachbaur** [comment]
Retest this please

File: tests/cpp/PixelStreamSegmentDecoderTests.cpp [hunk 1]
```diff
@@ -128,7 +130,7 @@ BOOST_AUTO_TEST_CASE( testImageSegmentationWithCompressionAndDecompression )
     size_t timeout = 0;
     while(decoder.isRunning())
     {
-        usleep(10);
+        boost::this_thread::sleep( boost::posix_time::microseconds( 10 ));
         if (++timeout >= 10)
             break;
     }
```
Comment (eile) [inline tests/cpp/PixelStreamSegmentDecoderTests.cpp:137]: fishy construct...

**Comment by eile** [comment]
+1

**Comment by dnachbaur** [comment]
Retest this please

**Comment by dnachbaur** [comment]
Retest this please

**Comment by dnachbaur** [comment]
Retest this please

**Pull request merged**